src/pybind/mgr/dashboard_v2/frontend/package.json

```diff
@@ -55,7 +55,8 @@
     "karma-jasmine-html-reporter": "^0.2.2",
     "karma-junit-reporter": "^1.2.0",
     "karma-phantomjs-launcher": "^1.0.4",
-    "node": "^8.9.4",
+    "node-bin-setup": "^1.0.6",
```

Comment:
I tested the current master in fedora 26 and the frontend build worked fine. I ran install-deps.sh script first and it installed NPM 5.7.1 and nodejs is at version v6.12.0. Then I ran `npm install` without problems

Reply:
On this system, I hadn't run install-deps because I knew I already had a recent enough npm+node to build the frontend.  It seems a bit unpleasant that the install-deps script wants to install a newer npm in order to install node, even though I already have a recent enough node :-/

Reply:
Which version of NPM do you have in that system?

Reply:
npm 3.10.10, node 6.12.0

Reply:
we are updating npm version in install-deps.sh script because npm < 5.0.0 cannot install nodejs as a local dependency.

Maybe the approach that @tspmelo referred in  https://github.com/ceph/ceph/pull/20752#discussion_r172813392 is the way to go, remove the nodejs as a local dependency and make sure in install-deps.sh script that the user has node > 6.9.0. That way npm does not need to be upgraded I guess.

Reply:
Just tested the frontend build without the local nodejs dependency in `package.json` in fedora 26 and everything works good with the npm and nodejs  versions installed by yum.